projects/elife.yaml

```diff
@@ -363,3 +371,48 @@ search:
         ports:
             - 22
             - 80
+
+bus:
+    default-branch: null
+    domain: null
+    int_domain: null
+    aws:
+        ec2: false
+        sns:
+            # bus-articles-prod, bus-articles-end2end, etc.
+            - bus-articles--{cluster}
```

Comment:
I would have `bus-articles--end2end` because this is an SNS topic, so it's a singleton; the same goes for a queue like `queue-name--end2end` which may be shared by two EC2 instances `search--end2end--1` and `search--end2end--2`.